source/faq/faq.rst

```diff
@@ -455,23 +438,26 @@ There is a very quick and useful tutorial in the :doc:`./../using-api-interface`
 I'm interested in integrating my application with dojot. How can I do it?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This should be pretty straightforward. There are two ways that your application
+This should be pretty straightforward. There are three ways that your application
 could be integrated with dojot:
 
   - **Retrieving historical data**: you might want to periodically read all
-    historical data related to a device. This can be done by using this API
-    (one side-note: all endpoints described in this apiary should be preceded
-    by ``/history/``).
-  - **Using flowbroker to pre-process data**: if you want to do something more, you
-    could use flows. They can help process and transform data so that they can
+    historical data related to a device. This can be done by using *history* API
+  - **Retrieving real time data**: you might want to in real time read data related to a device.
+    This can be done by using *kafka-ws*, a *websocket* based implementation.
+    To better understand how to use *kafka-ws* check :ref:`Kafka-WS Internal`.
+  - **Using flowbroker to pre-process data**: They can help process and
+    transform data so that they can
```

Comment:
Incomplete sentence. My suggestion is:
for more flexible ways of data manipulation, you can use flows. They can process/transform data so they can